web/src/topic_list.ts

```diff
@@ -616,6 +616,10 @@ export function initialize({
                 return;
             }
 
+            if (document.getSelection()?.type === "Range") {
+                e.preventDefault();
```

Comment:
Why are you preventing default here -- isn't the default behavior the selection that you're trying to enable in this PR?